js/download-page.js

```diff
@@ -6,7 +8,7 @@ $(document).ready(function() {
     });
 
     // Getting latest release information
-    $.getJSON(latest_versions_json, function(latest_pack) {
+    // $.getJSON(latest_versions_json, function(latest_pack) {
```

Comment:
we can remove this commented line?

Reply:
Body of this function had been already commented out. Hence, I just commented out function signature as well instead of removing it. Because, I am not sure why the body of the content was commented out.